src/IO/Observer/ReductionActions.hpp

```diff
@@ -17,6 +17,7 @@
 #include "IO/H5/File.hpp"
 #include "IO/Observer/ArrayComponentId.hpp"
 #include "IO/Observer/ObservationId.hpp"
+#include "IO/Observer/ObserverComponent.hpp"
```

Comment:
Should this be a forward declaration instead?

Reply:
I am forward declaring now.